Makefile

```diff
@@ -1,2 +1,21 @@
+THRIFT_CMD=./thrift-docker.sh
```

Comment:
the `go-thrift` one needs to be on root because of the directory restriction of docker. alternative is we copy the thrift file into the directory temporarily while running make. other rules do can live in subdirectory.

also when you have more than one directories you can no longer use the `%` rule (I don't believe you can use `py-%` in gnu make, but I could be wrong).

Reply:
I'm not sure I follow. The docker container should run here at the root level and so everything will be in the build context. 

Extend the % rule to use on multiple places:

```make
%:
    make -C lib/go/ $@
    make -C lib/py/ $@